### PR TITLE
Frontend tests

### PR DESCRIPTION
**Pull request by MicahSchiewe**
Will add more tests later

### CODE REVIEW

File: food-truck-frontend/src/components/manage-trucks/ManageTrucksComponent.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import Router from "next/router";
 
 import {Button, Card, CircularProgress, Container, Grid, Typography,} from "@material-ui/core";
 
-import api from "../../util/api";
+import { getTrucksForCurrentUser } from "../../api/Truck";
 import {TruckState} from "../TruckView";
 
 interface ManageTruckProps {}
```
Comment (nnazo) [inline food-truck-frontend/src/components/manage-trucks/ManageTrucksComponent.tsx]: Just noting here this component won't exist anymore after my PR is merged

File: food-truck-frontend/src/components/manage-trucks/__tests__/ManageTrucksComponent.test.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+jest.mock("../../../api/Truck", () => ({
+  getTrucksForCurrentUser: jest.fn().mockResolvedValue([mockMakeTruck()]),
+}));
+
+import { SNAPSHOT_TEST } from "../../../util/test_util/basic_tests";
+import ManageTrucksComponent from "../ManageTrucksComponent";
+
+SNAPSHOT_TEST("Snapshot manage trucks component", () => {
+  return (
+    <ManageTrucksComponent/>
```
Comment (nnazo) [inline food-truck-frontend/src/components/manage-trucks/__tests__/ManageTrucksComponent.test.tsx]: Same comment about component no longer existing

**Review by nnazo — APPROVED** [review]
LGTM, just need to remove that one component which is going to be gone soon

File: food-truck-frontend/src/api/RouteLocation.ts [hunk 3]
```diff
@@ -83,6 +87,19 @@ export const deleteRouteLocations = async (
...
+    if (response && response.data && response.data.isActive)
+      return true;
+    return false;
+  } catch (error) {
+    if (onFail)
+      onFail(error);
+  }
+  return undefined;
+}
+
```
Comment (KevinDeMars) [inline food-truck-frontend/src/api/RouteLocation.ts]: Maybe should be moved to api folder once api changes are merged

File: food-truck-frontend/src/api/Truck.ts [hunk 4]
```diff
@@ -49,6 +49,18 @@ export const getTruckById = async (truckId: number, onFail: (err: any) => void)
...
+    return (await api.request({
+        url: `/truck/owner`,
+        method: "GET",
+      })).data;
+  } catch (error) {
+    if (onFail)
+      onFail(error);
+  }
+}
+
```
Comment (KevinDeMars) [inline food-truck-frontend/src/api/Truck.ts]: See above

**Review by KevinDeMars — APPROVED** [review]
Looks good

**Review by nnazo — APPROVED** [review]
LGTM. Good job, Micah!

**Comment by MicahSchiewe** [comment]
Unable to reach 2 current approvals before code freeze

**Pull request closed without merging**